src/model/NumberColumn.ts

```diff
@@ -274,7 +274,7 @@ export default class NumberColumn extends ValueColumn<number> implements INumber
   private numberFormat: (n: number) => string = format('.2f');
 
   private currentStratifyThresholds: number[] = [];
-  private groupSortMethod: SortMethod = ADVANCED_SORT_METHOD.median;
+  private groupSortMethod: SortMethod = '';
```

Comment:
what happens if I specify a NumberColumn as my group sorting criteria via code? i.e. it tries to sort but there is no default sort method set anymore, thus will crash I assume.